From d95edbdd742c2f07478cd041489f69f217b70cbf Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Wed, 7 Sep 2022 15:20:01 +0800
Subject: [PATCH] sw64: fix incorrect white space use in macros
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: d95edbdd742c2f07478cd041489f69f217b70cbf
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Signed-off-by: Mao Minkai <maominkai@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/chip/chip3/cpufreq_debugfs.c |  18 +-
 arch/sw_64/include/asm/atomic.h         |   2 +-
 arch/sw_64/include/asm/barrier.h        |   2 +-
 arch/sw_64/include/asm/clock.h          |   2 +-
 arch/sw_64/include/asm/dma-mapping.h    |   2 +-
 arch/sw_64/include/asm/efi.h            |   2 +-
 arch/sw_64/include/asm/hcall.h          |   2 +-
 arch/sw_64/include/asm/insn.h           |   8 +-
 arch/sw_64/include/asm/kprobes.h        |   2 +-
 arch/sw_64/include/asm/kvm_host.h       |   2 +-
 arch/sw_64/include/asm/kvm_mmio.h       |   2 +-
 arch/sw_64/include/asm/msi.h            |   6 +-
 arch/sw_64/include/asm/numa.h           |   4 +-
 arch/sw_64/include/asm/pgtable-4level.h |   2 +-
 arch/sw_64/include/asm/pgtable.h        |   2 +-
 arch/sw_64/include/asm/platform.h       |   2 +-
 arch/sw_64/include/asm/stacktrace.h     |   2 +-
 arch/sw_64/include/asm/sw64_init.h      |   2 +-
 arch/sw_64/include/asm/syscall.h        |   2 +-
 arch/sw_64/include/asm/termios.h        |   2 +-
 arch/sw_64/include/asm/vcpu.h           |   6 +-
 arch/sw_64/include/uapi/asm/errno.h     | 220 ++++++++++++------------
 arch/sw_64/include/uapi/asm/ioctls.h    |   4 +-
 arch/sw_64/include/uapi/asm/kvm.h       |   2 +-
 arch/sw_64/include/uapi/asm/regdef.h    |  16 +-
 arch/sw_64/include/uapi/asm/termbits.h  | 128 +++++++-------
 arch/sw_64/kernel/acpi.c                |   4 +-
 arch/sw_64/kernel/pci_impl.h            |   2 +-
 arch/sw_64/kernel/setup.c               |   2 +-
 arch/sw_64/kernel/time.c                |   2 +-
 arch/sw_64/math-emu/math.c              |  10 +-
 31 files changed, 232 insertions(+), 232 deletions(-)

diff --git a/arch/sw_64/chip/chip3/cpufreq_debugfs.c b/arch/sw_64/chip/chip3/cpufreq_debugfs.c
index 13696360ef02..c58f1cee3907 100644
--- a/arch/sw_64/chip/chip3/cpufreq_debugfs.c
+++ b/arch/sw_64/chip/chip3/cpufreq_debugfs.c
@@ -7,15 +7,15 @@
 #include <asm/sw64io.h>
 #include <asm/debug.h>
 
-#define	CLK_PRT		0x1UL
-#define	CORE_CLK0_V	(0x1UL << 1)
-#define	CORE_CLK0_R	(0x1UL << 2)
-#define	CORE_CLK2_V	(0x1UL << 15)
-#define	CORE_CLK2_R	(0x1UL << 16)
-
-#define	CLK_LV1_SEL_PRT		0x1UL
-#define	CLK_LV1_SEL_MUXA	(0x1UL << 2)
-#define	CLK_LV1_SEL_MUXB	(0x1UL << 3)
+#define CLK_PRT		0x1UL
+#define CORE_CLK0_V	(0x1UL << 1)
+#define CORE_CLK0_R	(0x1UL << 2)
+#define CORE_CLK2_V	(0x1UL << 15)
+#define CORE_CLK2_R	(0x1UL << 16)
+
+#define CLK_LV1_SEL_PRT		0x1UL
+#define CLK_LV1_SEL_MUXA	(0x1UL << 2)
+#define CLK_LV1_SEL_MUXB	(0x1UL << 3)
 
 #define CORE_PLL0_CFG_SHIFT	4
 #define CORE_PLL2_CFG_SHIFT	18
diff --git a/arch/sw_64/include/asm/atomic.h b/arch/sw_64/include/asm/atomic.h
index 126417a1aeee..66f4437be103 100644
--- a/arch/sw_64/include/asm/atomic.h
+++ b/arch/sw_64/include/asm/atomic.h
@@ -336,7 +336,7 @@ ATOMIC_OPS(sub)
 #define atomic64_fetch_add_relaxed	atomic64_fetch_add_relaxed
 #define atomic64_fetch_sub_relaxed	atomic64_fetch_sub_relaxed
 
-#undef  ATOMIC_OPS
+#undef ATOMIC_OPS
 
 #define ATOMIC_OPS(op, asm)						\
 	ATOMIC_OP(op, asm)                                              \
diff --git a/arch/sw_64/include/asm/barrier.h b/arch/sw_64/include/asm/barrier.h
index c691038919cd..5f4a03d700c6 100644
--- a/arch/sw_64/include/asm/barrier.h
+++ b/arch/sw_64/include/asm/barrier.h
@@ -21,4 +21,4 @@
 
 #include <asm-generic/barrier.h>
 
-#endif		/* _ASM_SW64_BARRIER_H */
+#endif /* _ASM_SW64_BARRIER_H */
diff --git a/arch/sw_64/include/asm/clock.h b/arch/sw_64/include/asm/clock.h
index 88714eb08507..af6872ed9edb 100644
--- a/arch/sw_64/include/asm/clock.h
+++ b/arch/sw_64/include/asm/clock.h
@@ -53,4 +53,4 @@ void sw64_update_clockevents(unsigned long cpu, u32 freq);
 void sw64_store_policy(struct cpufreq_policy *policy);
 
 unsigned int __sw64_cpufreq_get(struct cpufreq_policy *policy);
-#endif				/* _ASM_SW64_CLOCK_H */
+#endif /* _ASM_SW64_CLOCK_H */
diff --git a/arch/sw_64/include/asm/dma-mapping.h b/arch/sw_64/include/asm/dma-mapping.h
index bb84690eabfe..37fce35b09d5 100644
--- a/arch/sw_64/include/asm/dma-mapping.h
+++ b/arch/sw_64/include/asm/dma-mapping.h
@@ -9,4 +9,4 @@ static inline const struct dma_map_ops *get_arch_dma_ops(struct bus_type *bus)
 	return dma_ops;
 }
 
-#endif	/* _ASM_SW64_DMA_MAPPING_H */
+#endif /* _ASM_SW64_DMA_MAPPING_H */
diff --git a/arch/sw_64/include/asm/efi.h b/arch/sw_64/include/asm/efi.h
index 2bc863e3b836..f061cae2fc8a 100644
--- a/arch/sw_64/include/asm/efi.h
+++ b/arch/sw_64/include/asm/efi.h
@@ -21,7 +21,7 @@ extern void efi_init(void);
 	__f(args);							\
 })
 
-#define	ARCH_EFI_IRQ_FLAGS_MASK		0x00000001
+#define ARCH_EFI_IRQ_FLAGS_MASK		0x00000001
 
 /* arch specific definitions used by the stub code */
 
diff --git a/arch/sw_64/include/asm/hcall.h b/arch/sw_64/include/asm/hcall.h
index b5438b477c87..65669e54c0f8 100644
--- a/arch/sw_64/include/asm/hcall.h
+++ b/arch/sw_64/include/asm/hcall.h
@@ -39,4 +39,4 @@ static inline unsigned long hcall(unsigned long hcall, unsigned long arg0,
 	return __r0;
 }
 
-#endif  /* _ASM_SW64_HCALL_H */
+#endif /* _ASM_SW64_HCALL_H */
diff --git a/arch/sw_64/include/asm/insn.h b/arch/sw_64/include/asm/insn.h
index 54a9a2026784..ec0efae3aed0 100644
--- a/arch/sw_64/include/asm/insn.h
+++ b/arch/sw_64/include/asm/insn.h
@@ -14,8 +14,8 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#ifndef	_ASM_SW64_INSN_H
-#define	_ASM_SW64_INSN_H
+#ifndef _ASM_SW64_INSN_H
+#define _ASM_SW64_INSN_H
 #include <linux/types.h>
 
 /* Register numbers */
@@ -28,7 +28,7 @@ enum {
 
 #define BR_MAX_DISP		0xfffff
 /* SW64 instructions are always 32 bits. */
-#define	SW64_INSN_SIZE		4
+#define SW64_INSN_SIZE		4
 
 #define ___SW64_RA(a)		(((a) & 0x1f) << 21)
 #define ___SW64_RB(b)		(((b) & 0x1f) << 16)
@@ -93,4 +93,4 @@ SW64_INSN(fbge,		0xf4000000, 0xfc000000);
 SW64_INSN(lldw,		0x20000000, 0xfc00f000);
 SW64_INSN(lldl,		0x20001000, 0xfc00f000);
 
-#endif	/* _ASM_SW64_INSN_H */
+#endif /* _ASM_SW64_INSN_H */
diff --git a/arch/sw_64/include/asm/kprobes.h b/arch/sw_64/include/asm/kprobes.h
index c19b961a19da..6b7e4548a8bd 100644
--- a/arch/sw_64/include/asm/kprobes.h
+++ b/arch/sw_64/include/asm/kprobes.h
@@ -19,7 +19,7 @@
 #include <asm/cacheflush.h>
 #include <asm/kdebug.h>
 
-#define	 __ARCH_WANT_KPROBES_INSN_SLOT
+#define __ARCH_WANT_KPROBES_INSN_SLOT
 
 struct kprobe;
 struct pt_regs;
diff --git a/arch/sw_64/include/asm/kvm_host.h b/arch/sw_64/include/asm/kvm_host.h
index 6d292c086347..835ccef7490f 100644
--- a/arch/sw_64/include/asm/kvm_host.h
+++ b/arch/sw_64/include/asm/kvm_host.h
@@ -124,4 +124,4 @@ static inline void kvm_arch_vcpu_blocking(struct kvm_vcpu *vcpu) {}
 static inline void kvm_arch_vcpu_unblocking(struct kvm_vcpu *vcpu) {}
 static inline void kvm_arch_vcpu_block_finish(struct kvm_vcpu *vcpu) {}
 
-#endif  /* _ASM_SW64_KVM_HOST_H */
+#endif /* _ASM_SW64_KVM_HOST_H */
diff --git a/arch/sw_64/include/asm/kvm_mmio.h b/arch/sw_64/include/asm/kvm_mmio.h
index 9ba31c91902f..c87b259e9395 100644
--- a/arch/sw_64/include/asm/kvm_mmio.h
+++ b/arch/sw_64/include/asm/kvm_mmio.h
@@ -14,4 +14,4 @@ int kvm_handle_mmio_return(struct kvm_vcpu *vcpu, struct kvm_run *run);
 int io_mem_abort(struct kvm_vcpu *vcpu, struct kvm_run *run,
 		 struct hcall_args *hargs);
 
-#endif	/* _ASM_SW64_KVM_MMIO_H */
+#endif /* _ASM_SW64_KVM_MMIO_H */
diff --git a/arch/sw_64/include/asm/msi.h b/arch/sw_64/include/asm/msi.h
index 079fac0d128e..ca5850eb5957 100644
--- a/arch/sw_64/include/asm/msi.h
+++ b/arch/sw_64/include/asm/msi.h
@@ -42,6 +42,6 @@ struct irq_alloc_info {
 	irq_hw_number_t         hwirq;
 };
 typedef struct irq_alloc_info msi_alloc_info_t;
-#endif	/* CONFIG_PCI_MSI_IRQ_DOMAIN */
-#endif	/* CONFIG_PCI_MSI */
-#endif	/* _ASM_SW64_MSI_H */
+#endif /* CONFIG_PCI_MSI_IRQ_DOMAIN */
+#endif /* CONFIG_PCI_MSI */
+#endif /* _ASM_SW64_MSI_H */
diff --git a/arch/sw_64/include/asm/numa.h b/arch/sw_64/include/asm/numa.h
index 4ea8b8de248a..930be7831813 100644
--- a/arch/sw_64/include/asm/numa.h
+++ b/arch/sw_64/include/asm/numa.h
@@ -30,6 +30,6 @@ static inline void  set_cpuid_to_node(int cpuid, s16 node)
 	__cpuid_to_node[cpuid] = node;
 }
 
-#endif	/* CONFIG_NUMA */
+#endif /* CONFIG_NUMA */
 
-#endif	/* _ASM_SW64_NUMA_H */
+#endif /* _ASM_SW64_NUMA_H */
diff --git a/arch/sw_64/include/asm/pgtable-4level.h b/arch/sw_64/include/asm/pgtable-4level.h
index 8c45f441c520..719e2c5377e3 100644
--- a/arch/sw_64/include/asm/pgtable-4level.h
+++ b/arch/sw_64/include/asm/pgtable-4level.h
@@ -2,7 +2,7 @@
 #ifndef _ASM_SW64_PGTABLE_4LEVEL_H
 #define _ASM_SW64_PGTABLE_4LEVEL_H
 
-#ifdef	__KERNEL__
+#ifdef __KERNEL__
 #ifndef __ASSEMBLY__
 /*
  * These are used to make use of C type-checking..
diff --git a/arch/sw_64/include/asm/pgtable.h b/arch/sw_64/include/asm/pgtable.h
index 76c782baf242..b451bc94e737 100644
--- a/arch/sw_64/include/asm/pgtable.h
+++ b/arch/sw_64/include/asm/pgtable.h
@@ -544,7 +544,7 @@ static inline void pmdp_set_wrprotect(struct mm_struct *mm,
 
 #define mk_pmd(page, prot)	pfn_pmd(page_to_pfn(page), (prot))
 
-#define  __HAVE_ARCH_PMDP_SET_ACCESS_FLAGS
+#define __HAVE_ARCH_PMDP_SET_ACCESS_FLAGS
 extern int pmdp_set_access_flags(struct vm_area_struct *vma,
 				 unsigned long address, pmd_t *pmdp,
 				 pmd_t entry, int dirty);
diff --git a/arch/sw_64/include/asm/platform.h b/arch/sw_64/include/asm/platform.h
index 318b6ca732cd..59418bba9de7 100644
--- a/arch/sw_64/include/asm/platform.h
+++ b/arch/sw_64/include/asm/platform.h
@@ -14,4 +14,4 @@ extern struct sw64_platform_ops *sw64_platform;
 
 extern struct sw64_platform_ops xuelang_ops;
 
-#endif  /* _ASM_SW64_PLATFORM_H */
+#endif /* _ASM_SW64_PLATFORM_H */
diff --git a/arch/sw_64/include/asm/stacktrace.h b/arch/sw_64/include/asm/stacktrace.h
index ed691a72573b..958c9892fd6d 100644
--- a/arch/sw_64/include/asm/stacktrace.h
+++ b/arch/sw_64/include/asm/stacktrace.h
@@ -69,4 +69,4 @@ static inline bool on_accessible_stack(struct task_struct *tsk,
 	return false;
 }
 
-#endif	/* _ASM_SW64_STACKTRACE_H */
+#endif /* _ASM_SW64_STACKTRACE_H */
diff --git a/arch/sw_64/include/asm/sw64_init.h b/arch/sw_64/include/asm/sw64_init.h
index 2d9140605d0b..9f16bdf2a61a 100644
--- a/arch/sw_64/include/asm/sw64_init.h
+++ b/arch/sw_64/include/asm/sw64_init.h
@@ -43,4 +43,4 @@ extern struct sw64_chip_init_ops *sw64_chip_init;
 
 DECLARE_PER_CPU(unsigned long, hard_node_id);
 
-#endif    /* _ASM_SW64_INIT_H */
+#endif /* _ASM_SW64_INIT_H */
diff --git a/arch/sw_64/include/asm/syscall.h b/arch/sw_64/include/asm/syscall.h
index 4b784c3d846b..0b1556a460b4 100644
--- a/arch/sw_64/include/asm/syscall.h
+++ b/arch/sw_64/include/asm/syscall.h
@@ -72,4 +72,4 @@ static inline int syscall_get_arch(struct task_struct *task)
 	return AUDIT_ARCH_SW64;
 }
 
-#endif  /* _ASM_SW64_SYSCALL_H */
+#endif /* _ASM_SW64_SYSCALL_H */
diff --git a/arch/sw_64/include/asm/termios.h b/arch/sw_64/include/asm/termios.h
index ef509946675a..9849dd9b58bf 100644
--- a/arch/sw_64/include/asm/termios.h
+++ b/arch/sw_64/include/asm/termios.h
@@ -78,4 +78,4 @@
 #define kernel_termios_to_user_termios(u, k) \
 	copy_to_user(u, k, sizeof(struct termios))
 
-#endif	/* _ASM_SW64_TERMIOS_H */
+#endif /* _ASM_SW64_TERMIOS_H */
diff --git a/arch/sw_64/include/asm/vcpu.h b/arch/sw_64/include/asm/vcpu.h
index 476c396c5aa4..dfefb9dc8651 100644
--- a/arch/sw_64/include/asm/vcpu.h
+++ b/arch/sw_64/include/asm/vcpu.h
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 #ifndef _ASM_SW64_VCPU_H
-#define  _ASM_SW64_VCPU_H
+#define _ASM_SW64_VCPU_H
 
 #ifndef __ASSEMBLY__
 
@@ -43,5 +43,5 @@ struct vcpucb {
 	unsigned long vcpu_irq_vector;
 };
 
-#endif  /* __ASSEMBLY__ */
-#endif  /* _ASM_SW64_VCPU_H */
+#endif /* __ASSEMBLY__ */
+#endif /* _ASM_SW64_VCPU_H */
diff --git a/arch/sw_64/include/uapi/asm/errno.h b/arch/sw_64/include/uapi/asm/errno.h
index 0d8438f6bd40..94e1eab5d003 100644
--- a/arch/sw_64/include/uapi/asm/errno.h
+++ b/arch/sw_64/include/uapi/asm/errno.h
@@ -4,124 +4,124 @@
 
 #include <asm-generic/errno-base.h>
 
-#undef	EAGAIN			/* 11 in errno-base.h */
-
-#define	EDEADLK		11	/* Resource deadlock would occur */
-
-#define	EAGAIN		35	/* Try again */
-#define	EWOULDBLOCK	EAGAIN	/* Operation would block */
-#define	EINPROGRESS	36	/* Operation now in progress */
-#define	EALREADY	37	/* Operation already in progress */
-#define	ENOTSOCK	38	/* Socket operation on non-socket */
-#define	EDESTADDRREQ	39	/* Destination address required */
-#define	EMSGSIZE	40	/* Message too long */
-#define	EPROTOTYPE	41	/* Protocol wrong type for socket */
-#define	ENOPROTOOPT	42	/* Protocol not available */
-#define	EPROTONOSUPPORT	43	/* Protocol not supported */
-#define	ESOCKTNOSUPPORT	44	/* Socket type not supported */
-#define	EOPNOTSUPP	45	/* Operation not supported on transport endpoint */
-#define	EPFNOSUPPORT	46	/* Protocol family not supported */
-#define	EAFNOSUPPORT	47	/* Address family not supported by protocol */
-#define	EADDRINUSE	48	/* Address already in use */
-#define	EADDRNOTAVAIL	49	/* Cannot assign requested address */
-#define	ENETDOWN	50	/* Network is down */
-#define	ENETUNREACH	51	/* Network is unreachable */
-#define	ENETRESET	52	/* Network dropped connection because of reset */
-#define	ECONNABORTED	53	/* Software caused connection abort */
-#define	ECONNRESET	54	/* Connection reset by peer */
-#define	ENOBUFS		55	/* No buffer space available */
-#define	EISCONN		56	/* Transport endpoint is already connected */
-#define	ENOTCONN	57	/* Transport endpoint is not connected */
-#define	ESHUTDOWN	58	/* Cannot send after transport endpoint shutdown */
-#define	ETOOMANYREFS	59	/* Too many references: cannot splice */
-#define	ETIMEDOUT	60	/* Connection timed out */
-#define	ECONNREFUSED	61	/* Connection refused */
-#define	ELOOP		62	/* Too many symbolic links encountered */
-#define	ENAMETOOLONG	63	/* File name too long */
-#define	EHOSTDOWN	64	/* Host is down */
-#define	EHOSTUNREACH	65	/* No route to host */
-#define	ENOTEMPTY	66	/* Directory not empty */
-
-#define	EUSERS		68	/* Too many users */
-#define	EDQUOT		69	/* Quota exceeded */
-#define	ESTALE		70	/* Stale NFS file handle */
-#define	EREMOTE		71	/* Object is remote */
-
-#define	ENOLCK		77	/* No record locks available */
-#define	ENOSYS		78	/* Function not implemented */
-
-#define	ENOMSG		80	/* No message of desired type */
-#define	EIDRM		81	/* Identifier removed */
-#define	ENOSR		82	/* Out of streams resources */
-#define	ETIME		83	/* Timer expired */
-#define	EBADMSG		84	/* Not a data message */
-#define	EPROTO		85	/* Protocol error */
-#define	ENODATA		86	/* No data available */
-#define	ENOSTR		87	/* Device not a stream */
-
-#define	ENOPKG		92	/* Package not installed */
-
-#define	EILSEQ		116	/* Illegal byte sequence */
+#undef EAGAIN			/* 11 in errno-base.h */
+
+#define EDEADLK		11	/* Resource deadlock would occur */
+
+#define EAGAIN		35	/* Try again */
+#define EWOULDBLOCK	EAGAIN	/* Operation would block */
+#define EINPROGRESS	36	/* Operation now in progress */
+#define EALREADY	37	/* Operation already in progress */
+#define ENOTSOCK	38	/* Socket operation on non-socket */
+#define EDESTADDRREQ	39	/* Destination address required */
+#define EMSGSIZE	40	/* Message too long */
+#define EPROTOTYPE	41	/* Protocol wrong type for socket */
+#define ENOPROTOOPT	42	/* Protocol not available */
+#define EPROTONOSUPPORT	43	/* Protocol not supported */
+#define ESOCKTNOSUPPORT	44	/* Socket type not supported */
+#define EOPNOTSUPP	45	/* Operation not supported on transport endpoint */
+#define EPFNOSUPPORT	46	/* Protocol family not supported */
+#define EAFNOSUPPORT	47	/* Address family not supported by protocol */
+#define EADDRINUSE	48	/* Address already in use */
+#define EADDRNOTAVAIL	49	/* Cannot assign requested address */
+#define ENETDOWN	50	/* Network is down */
+#define ENETUNREACH	51	/* Network is unreachable */
+#define ENETRESET	52	/* Network dropped connection because of reset */
+#define ECONNABORTED	53	/* Software caused connection abort */
+#define ECONNRESET	54	/* Connection reset by peer */
+#define ENOBUFS		55	/* No buffer space available */
+#define EISCONN		56	/* Transport endpoint is already connected */
+#define ENOTCONN	57	/* Transport endpoint is not connected */
+#define ESHUTDOWN	58	/* Cannot send after transport endpoint shutdown */
+#define ETOOMANYREFS	59	/* Too many references: cannot splice */
+#define ETIMEDOUT	60	/* Connection timed out */
+#define ECONNREFUSED	61	/* Connection refused */
+#define ELOOP		62	/* Too many symbolic links encountered */
+#define ENAMETOOLONG	63	/* File name too long */
+#define EHOSTDOWN	64	/* Host is down */
+#define EHOSTUNREACH	65	/* No route to host */
+#define ENOTEMPTY	66	/* Directory not empty */
+
+#define EUSERS		68	/* Too many users */
+#define EDQUOT		69	/* Quota exceeded */
+#define ESTALE		70	/* Stale NFS file handle */
+#define EREMOTE		71	/* Object is remote */
+
+#define ENOLCK		77	/* No record locks available */
+#define ENOSYS		78	/* Function not implemented */
+
+#define ENOMSG		80	/* No message of desired type */
+#define EIDRM		81	/* Identifier removed */
+#define ENOSR		82	/* Out of streams resources */
+#define ETIME		83	/* Timer expired */
+#define EBADMSG		84	/* Not a data message */
+#define EPROTO		85	/* Protocol error */
+#define ENODATA		86	/* No data available */
+#define ENOSTR		87	/* Device not a stream */
+
+#define ENOPKG		92	/* Package not installed */
+
+#define EILSEQ		116	/* Illegal byte sequence */
 
 /* The following are just random noise.. */
-#define	ECHRNG		88	/* Channel number out of range */
-#define	EL2NSYNC	89	/* Level 2 not synchronized */
-#define	EL3HLT		90	/* Level 3 halted */
-#define	EL3RST		91	/* Level 3 reset */
-
-#define	ELNRNG		93	/* Link number out of range */
-#define	EUNATCH		94	/* Protocol driver not attached */
-#define	ENOCSI		95	/* No CSI structure available */
-#define	EL2HLT		96	/* Level 2 halted */
-#define	EBADE		97	/* Invalid exchange */
-#define	EBADR		98	/* Invalid request descriptor */
-#define	EXFULL		99	/* Exchange full */
-#define	ENOANO		100	/* No anode */
-#define	EBADRQC		101	/* Invalid request code */
-#define	EBADSLT		102	/* Invalid slot */
-
-#define	EDEADLOCK	EDEADLK
-
-#define	EBFONT		104	/* Bad font file format */
-#define	ENONET		105	/* Machine is not on the network */
-#define	ENOLINK		106	/* Link has been severed */
-#define	EADV		107	/* Advertise error */
-#define	ESRMNT		108	/* Srmount error */
-#define	ECOMM		109	/* Communication error on send */
-#define	EMULTIHOP	110	/* Multihop attempted */
-#define	EDOTDOT		111	/* RFS specific error */
-#define	EOVERFLOW	112	/* Value too large for defined data type */
-#define	ENOTUNIQ	113	/* Name not unique on network */
-#define	EBADFD		114	/* File descriptor in bad state */
-#define	EREMCHG		115	/* Remote address changed */
-
-#define	EUCLEAN		117	/* Structure needs cleaning */
-#define	ENOTNAM		118	/* Not a XENIX named type file */
-#define	ENAVAIL		119	/* No XENIX semaphores available */
-#define	EISNAM		120	/* Is a named type file */
-#define	EREMOTEIO	121	/* Remote I/O error */
-
-#define	ELIBACC		122	/* Can not access a needed shared library */
-#define	ELIBBAD		123	/* Accessing a corrupted shared library */
-#define	ELIBSCN		124	/* .lib section in a.out corrupted */
-#define	ELIBMAX		125	/* Attempting to link in too many shared libraries */
-#define	ELIBEXEC	126	/* Cannot exec a shared library directly */
-#define	ERESTART	127	/* Interrupted system call should be restarted */
-#define	ESTRPIPE	128	/* Streams pipe error */
+#define ECHRNG		88	/* Channel number out of range */
+#define EL2NSYNC	89	/* Level 2 not synchronized */
+#define EL3HLT		90	/* Level 3 halted */
+#define EL3RST		91	/* Level 3 reset */
+
+#define ELNRNG		93	/* Link number out of range */
+#define EUNATCH		94	/* Protocol driver not attached */
+#define ENOCSI		95	/* No CSI structure available */
+#define EL2HLT		96	/* Level 2 halted */
+#define EBADE		97	/* Invalid exchange */
+#define EBADR		98	/* Invalid request descriptor */
+#define EXFULL		99	/* Exchange full */
+#define ENOANO		100	/* No anode */
+#define EBADRQC		101	/* Invalid request code */
+#define EBADSLT		102	/* Invalid slot */
+
+#define EDEADLOCK	EDEADLK
+
+#define EBFONT		104	/* Bad font file format */
+#define ENONET		105	/* Machine is not on the network */
+#define ENOLINK		106	/* Link has been severed */
+#define EADV		107	/* Advertise error */
+#define ESRMNT		108	/* Srmount error */
+#define ECOMM		109	/* Communication error on send */
+#define EMULTIHOP	110	/* Multihop attempted */
+#define EDOTDOT		111	/* RFS specific error */
+#define EOVERFLOW	112	/* Value too large for defined data type */
+#define ENOTUNIQ	113	/* Name not unique on network */
+#define EBADFD		114	/* File descriptor in bad state */
+#define EREMCHG		115	/* Remote address changed */
+
+#define EUCLEAN		117	/* Structure needs cleaning */
+#define ENOTNAM		118	/* Not a XENIX named type file */
+#define ENAVAIL		119	/* No XENIX semaphores available */
+#define EISNAM		120	/* Is a named type file */
+#define EREMOTEIO	121	/* Remote I/O error */
+
+#define ELIBACC		122	/* Can not access a needed shared library */
+#define ELIBBAD		123	/* Accessing a corrupted shared library */
+#define ELIBSCN		124	/* .lib section in a.out corrupted */
+#define ELIBMAX		125	/* Attempting to link in too many shared libraries */
+#define ELIBEXEC	126	/* Cannot exec a shared library directly */
+#define ERESTART	127	/* Interrupted system call should be restarted */
+#define ESTRPIPE	128	/* Streams pipe error */
 
 #define ENOMEDIUM	129	/* No medium found */
 #define EMEDIUMTYPE	130	/* Wrong medium type */
-#define	ECANCELED	131	/* Operation Cancelled */
-#define	ENOKEY		132	/* Required key not available */
-#define	EKEYEXPIRED	133	/* Key has expired */
-#define	EKEYREVOKED	134	/* Key has been revoked */
-#define	EKEYREJECTED	135	/* Key was rejected by service */
+#define ECANCELED	131	/* Operation Cancelled */
+#define ENOKEY		132	/* Required key not available */
+#define EKEYEXPIRED	133	/* Key has expired */
+#define EKEYREVOKED	134	/* Key has been revoked */
+#define EKEYREJECTED	135	/* Key was rejected by service */
 
 /* for robust mutexes */
-#define	EOWNERDEAD	136	/* Owner died */
-#define	ENOTRECOVERABLE	137	/* State not recoverable */
+#define EOWNERDEAD	136	/* Owner died */
+#define ENOTRECOVERABLE	137	/* State not recoverable */
 
-#define	ERFKILL		138	/* Operation not possible due to RF-kill */
+#define ERFKILL		138	/* Operation not possible due to RF-kill */
 
 #define EHWPOISON	139	/* Memory page has hardware error */
 
diff --git a/arch/sw_64/include/uapi/asm/ioctls.h b/arch/sw_64/include/uapi/asm/ioctls.h
index db8e456290e6..751a07fd0303 100644
--- a/arch/sw_64/include/uapi/asm/ioctls.h
+++ b/arch/sw_64/include/uapi/asm/ioctls.h
@@ -34,8 +34,8 @@
 
 #define TIOCSWINSZ	_IOW('t', 103, struct winsize)
 #define TIOCGWINSZ	_IOR('t', 104, struct winsize)
-#define	TIOCSTART	_IO('t', 110)		/* start output, like ^Q */
-#define	TIOCSTOP	_IO('t', 111)		/* stop output, like ^S */
+#define TIOCSTART	_IO('t', 110)		/* start output, like ^Q */
+#define TIOCSTOP	_IO('t', 111)		/* stop output, like ^S */
 #define TIOCOUTQ	_IOR('t', 115, int)	/* output queue size */
 
 #define TIOCGLTC	_IOR('t', 116, struct ltchars)
diff --git a/arch/sw_64/include/uapi/asm/kvm.h b/arch/sw_64/include/uapi/asm/kvm.h
index 126c2a1d7411..254d6cbf1eb1 100644
--- a/arch/sw_64/include/uapi/asm/kvm.h
+++ b/arch/sw_64/include/uapi/asm/kvm.h
@@ -114,4 +114,4 @@ struct kvm_sync_regs {
 struct kvm_sregs {
 };
 
-#endif  /* _UAPI_ASM_SW64_KVM_H */
+#endif /* _UAPI_ASM_SW64_KVM_H */
diff --git a/arch/sw_64/include/uapi/asm/regdef.h b/arch/sw_64/include/uapi/asm/regdef.h
index ad4475b79435..7460a987c726 100644
--- a/arch/sw_64/include/uapi/asm/regdef.h
+++ b/arch/sw_64/include/uapi/asm/regdef.h
@@ -13,14 +13,14 @@
 #define t6	$7
 #define t7	$8
 
-#define	s0	$9	/* saved-registers (callee-saved registers) */
-#define	s1	$10
-#define	s2	$11
-#define	s3	$12
-#define	s4	$13
-#define	s5	$14
-#define	s6	$15
-#define	fp	s6	/* frame-pointer (s6 in frame-less procedures) */
+#define s0	$9	/* saved-registers (callee-saved registers) */
+#define s1	$10
+#define s2	$11
+#define s3	$12
+#define s4	$13
+#define s5	$14
+#define s6	$15
+#define fp	s6	/* frame-pointer (s6 in frame-less procedures) */
 
 #define a0	$16	/* argument registers (caller-saved) */
 #define a1	$17
diff --git a/arch/sw_64/include/uapi/asm/termbits.h b/arch/sw_64/include/uapi/asm/termbits.h
index 83de6ff63234..aaadb1d54f92 100644
--- a/arch/sw_64/include/uapi/asm/termbits.h
+++ b/arch/sw_64/include/uapi/asm/termbits.h
@@ -87,73 +87,73 @@ struct ktermios {
 #define OFILL		00000100
 #define OFDEL		00000200
 #define NLDLY		00001400
-#define   NL0		00000000
-#define   NL1		00000400
-#define   NL2		00001000
-#define   NL3		00001400
+#define NL0		00000000
+#define NL1		00000400
+#define NL2		00001000
+#define NL3		00001400
 #define TABDLY		00006000
-#define   TAB0		00000000
-#define   TAB1		00002000
-#define   TAB2		00004000
-#define   TAB3		00006000
-#define	CRDLY		00030000
-#define   CR0		00000000
-#define   CR1		00010000
-#define   CR2		00020000
-#define   CR3		00030000
+#define TAB0		00000000
+#define TAB1		00002000
+#define TAB2		00004000
+#define TAB3		00006000
+#define CRDLY		00030000
+#define CR0		00000000
+#define CR1		00010000
+#define CR2		00020000
+#define CR3		00030000
 #define FFDLY		00040000
-#define   FF0		00000000
-#define   FF1		00040000
+#define FF0		00000000
+#define FF1		00040000
 #define BSDLY		00100000
-#define   BS0		00000000
-#define   BS1		00100000
+#define BS0		00000000
+#define BS1		00100000
 #define VTDLY		00200000
-#define   VT0		00000000
-#define   VT1		00200000
+#define VT0		00000000
+#define VT1		00200000
 #define XTABS		01000000	/* Hmm.. Linux/i386 considers this part of TABDLY.. */
 
 /* c_cflag bit meaning */
 #define CBAUD		0000037
-#define  B0		0000000		/* hang up */
-#define  B50		0000001
-#define  B75		0000002
-#define  B110		0000003
-#define  B134		0000004
-#define  B150		0000005
-#define  B200		0000006
-#define  B300		0000007
-#define  B600		0000010
-#define  B1200		0000011
-#define  B1800		0000012
-#define  B2400		0000013
-#define  B4800		0000014
-#define  B9600		0000015
-#define  B19200		0000016
-#define  B38400		0000017
+#define B0		0000000		/* hang up */
+#define B50		0000001
+#define B75		0000002
+#define B110		0000003
+#define B134		0000004
+#define B150		0000005
+#define B200		0000006
+#define B300		0000007
+#define B600		0000010
+#define B1200		0000011
+#define B1800		0000012
+#define B2400		0000013
+#define B4800		0000014
+#define B9600		0000015
+#define B19200		0000016
+#define B38400		0000017
 #define EXTA		B19200
 #define EXTB		B38400
 #define CBAUDEX		0000000
-#define  B57600		00020
-#define  B115200	00021
-#define  B230400	00022
-#define  B460800	00023
-#define  B500000	00024
-#define  B576000	00025
-#define  B921600	00026
-#define  B1000000	00027
-#define  B1152000	00030
-#define  B1500000	00031
-#define  B2000000	00032
-#define  B2500000	00033
-#define  B3000000	00034
-#define  B3500000	00035
-#define  B4000000	00036
+#define B57600		00020
+#define B115200	00021
+#define B230400	00022
+#define B460800	00023
+#define B500000	00024
+#define B576000	00025
+#define B921600	00026
+#define B1000000	00027
+#define B1152000	00030
+#define B1500000	00031
+#define B2000000	00032
+#define B2500000	00033
+#define B3000000	00034
+#define B3500000	00035
+#define B4000000	00036
 
 #define CSIZE		00001400
-#define   CS5		00000000
-#define   CS6		00000400
-#define   CS7		00001000
-#define   CS8		00001400
+#define CS5		00000000
+#define CS6		00000400
+#define CS7		00001000
+#define CS8		00001400
 
 #define CSTOPB		00002000
 #define CREAD		00004000
@@ -184,19 +184,19 @@ struct ktermios {
 #define EXTPROC		0x10000000
 
 /* Values for the ACTION argument to `tcflow'.  */
-#define	TCOOFF		0
-#define	TCOON		1
-#define	TCIOFF		2
-#define	TCION		3
+#define TCOOFF		0
+#define TCOON		1
+#define TCIOFF		2
+#define TCION		3
 
 /* Values for the QUEUE_SELECTOR argument to `tcflush'.  */
-#define	TCIFLUSH	0
-#define	TCOFLUSH	1
-#define	TCIOFLUSH	2
+#define TCIFLUSH	0
+#define TCOFLUSH	1
+#define TCIOFLUSH	2
 
 /* Values for the OPTIONAL_ACTIONS argument to `tcsetattr'.  */
-#define	TCSANOW		0
-#define	TCSADRAIN	1
-#define	TCSAFLUSH	2
+#define TCSANOW		0
+#define TCSADRAIN	1
+#define TCSAFLUSH	2
 
 #endif /* _UAPI_ASM_SW64_TERMBITS_H */
diff --git a/arch/sw_64/kernel/acpi.c b/arch/sw_64/kernel/acpi.c
index a0b5c4a57a07..61f2948f1781 100644
--- a/arch/sw_64/kernel/acpi.c
+++ b/arch/sw_64/kernel/acpi.c
@@ -97,7 +97,7 @@ int acpi_unmap_lsapic(int cpu)
 	return 0;
 }
 EXPORT_SYMBOL(acpi_unmap_lsapic);
-#endif				/* CONFIG_ACPI_HOTPLUG_CPU */
+#endif /* CONFIG_ACPI_HOTPLUG_CPU */
 
 u8 acpi_checksum(u8 *table, u32 length)
 {
@@ -361,7 +361,7 @@ int acpi_unmap_cpu(int cpu)
 	return 0;
 }
 EXPORT_SYMBOL(acpi_unmap_cpu);
-#endif				/* CONFIG_ACPI_HOTPLUG_CPU */
+#endif /* CONFIG_ACPI_HOTPLUG_CPU */
 
 void __init acpi_boot_table_init(void)
 
diff --git a/arch/sw_64/kernel/pci_impl.h b/arch/sw_64/kernel/pci_impl.h
index 6025145cb1c5..41d83a41db56 100644
--- a/arch/sw_64/kernel/pci_impl.h
+++ b/arch/sw_64/kernel/pci_impl.h
@@ -4,7 +4,7 @@
  * with the PCI initialization routines.
  */
 #ifndef _SW64_KERNEL_PCI_IMPL_H
-#define	_SW64_KERNEL_PCI_IMPL_H
+#define _SW64_KERNEL_PCI_IMPL_H
 
 #include <asm/sw64io.h>
 
diff --git a/arch/sw_64/kernel/setup.c b/arch/sw_64/kernel/setup.c
index 2d2a8c6d4b4e..f68d93b5a7b7 100644
--- a/arch/sw_64/kernel/setup.c
+++ b/arch/sw_64/kernel/setup.c
@@ -785,7 +785,7 @@ setup_arch(char **cmdline_p)
 		strlcat(boot_command_line, " ", COMMAND_LINE_SIZE);
 		strlcat(boot_command_line, builtin_cmdline, COMMAND_LINE_SIZE);
 	}
-#endif	/* CMDLINE_EXTEND */
+#endif /* CMDLINE_EXTEND */
 #endif
 	if (IS_ENABLED(CONFIG_SW64_CHIP3_ASIC_DEBUG) &&
 			IS_ENABLED(CONFIG_SW64_CHIP3)) {
diff --git a/arch/sw_64/kernel/time.c b/arch/sw_64/kernel/time.c
index 3aa55c886e38..32690e78849b 100644
--- a/arch/sw_64/kernel/time.c
+++ b/arch/sw_64/kernel/time.c
@@ -60,7 +60,7 @@ void arch_irq_work_raise(void)
 	set_irq_work_pending_flag();
 }
 
-#else  /* CONFIG_IRQ_WORK */
+#else /* CONFIG_IRQ_WORK */
 
 #define test_irq_work_pending()      0
 #define clear_irq_work_pending()
diff --git a/arch/sw_64/math-emu/math.c b/arch/sw_64/math-emu/math.c
index 9f281d82ad83..6da3aadcff88 100644
--- a/arch/sw_64/math-emu/math.c
+++ b/arch/sw_64/math-emu/math.c
@@ -188,14 +188,14 @@ void write_fp_reg_s(unsigned long reg, unsigned long val_p0,
 		unsigned long p1, unsigned long p2, unsigned long p3);
 void write_fp_reg_d(unsigned long reg, unsigned long val_p0,
 		unsigned long p1, unsigned long p2, unsigned long p3);
-#define	LOW_64_WORKING	1
+#define LOW_64_WORKING	1
 #define HIGH_64_WORKING	2
 
 /*
  * End for sw64
  */
 
-#define	OPC_HMC		0x00
+#define OPC_HMC		0x00
 #define OPC_INTA	0x10
 #define OPC_INTL	0x11
 #define OPC_INTS	0x12
@@ -205,7 +205,7 @@ void write_fp_reg_d(unsigned long reg, unsigned long val_p0,
 #define OPC_FLTI	0x16
 #define OPC_FLTL	0x17
 #define OPC_MISC	0x18
-#define	OPC_JSR		0x1a
+#define OPC_JSR		0x1a
 
 #define FOP_SRC_S	0
 #define FOP_SRC_T	2
@@ -295,9 +295,9 @@ void cleanup_module(void)
 	sw64_fp_emul = save_emul;
 }
 
-#undef  sw64_fp_emul_imprecise
+#undef sw64_fp_emul_imprecise
 #define sw64_fp_emul_imprecise		do_sw_fp_emul_imprecise
-#undef  sw64_fp_emul
+#undef sw64_fp_emul
 #define sw64_fp_emul			do_sw_fp_emul
 
 #endif /* MODULE */
-- 
2.33.0

